Source/WebCore/html/parser/HTMLPreloadScanner.cpp

```diff
@@ -248,6 +248,8 @@ class TokenPreloadScanner::StartTagScanner {
                     m_scriptType = ScriptType::Module;
                 else if (equalLettersIgnoringASCIICase(attributeValue, "importmap"_s))
                     m_scriptType = ScriptType::ImportMap;
+                else if (!MIMETypeRegistry::isSupportedJavaScriptMIMEType(attributeValue.trim(deprecatedIsSpaceOrNewline).toStringWithoutCopying()))
```

Comment:
Why this deprecated definition of spaces? If this needs to match a test elsewhere maybe we should have paired comments to say they must be the same, or a shared function.

Reply:
I matched the logic in HTMLScriptElement. I will look into sharing.